src/Raven.Server/Documents/Queries/QueryMetadata.cs

```diff
@@ -486,6 +502,55 @@ private void HandleSelectFunctionBody(BlittableJsonReaderObject parameters)
             SelectFields = new[] { SelectField.CreateMethodCall(name, null, args) };
         }
 
+        private void AddToCounterIncludes(CounterIncludesField counterIncludes, MethodExpression expression, BlittableJsonReaderObject parameters)
+        {
+            string sourcePath = null;
+            var start = 0;
+            if (expression.Arguments.Count > 0 &&
+                expression.Arguments[0] is FieldExpression fe)
+            {
+                start = 1;
+
+                if (Query.From.Alias?.Value != fe.FieldValue &&
+                    RootAliasPaths.TryGetValue(fe.FieldValue, out var value))
+                {
+                    sourcePath = value.PropertyPath;
+                }
+            }
+
+            if (start == expression.Arguments.Count)
+            {
+                counterIncludes.Counters[sourcePath ?? string.Empty] = new HashSet<string>();
+                return;
+            }
+
+            for (var index = start; index < expression.Arguments.Count; index++)
```

Comment:
IIRC `QueryBuilder.GetValues` is doing the same thing.

Reply:
fixed